lib/assessment.js

```diff
@@ -8,23 +8,27 @@
 // Question 1
 // Write a callback for reduce that selects
 // the minimum value in the array
-var min = function min() {
-
+var min = function min(previousValue, currentValue) {
+  return previousValue < currentValue? previousValue : currentValue;
 };
 
+
 // Question 2
 // Write a callback for reduce that produces
 // the sum of the values in the array
-var sum = function sum() {
-
+var sum = function sum(previousValue, currentValue) {
+  return previousValue += currentValue;
```

Comment:
Technically will work, but `return previousValue + currentValue` would be more clear.